.github/workflows/docker-publish.yml

```diff
@@ -2,7 +2,7 @@ name: docker-publish.yml
 
 on:
   push:
-    branches: [main, master, gha-test-*]
+    branches: [auto, release-*]
```

Comment:
turns out it's helpful to publish docker hub release when we're on the release branch.

Reply:
there's an argument to made to add canary here.